Makefile

```diff
@@ -1,5 +1,11 @@
 run-tests:
 	pipenv run pytest test_pipenv.py
+run-tox:
+	pipenv run tox
```

Comment:
The same thing is done on the previous makefile target (run-tests). I thought if it's good enough to run pytest, then it should be ok for running tox.